examples/rust/pipe-ping-pong.rs

```diff
@@ -222,58 +206,38 @@ impl PipeServer {
 // Drop implementation for the pipe server.
 impl Drop for PipeServer {
     fn drop(&mut self) {
-        self.close(self.pipeqd_rx);
-        self.close(self.pipeqd_tx);
+        self.close(self.pipeqd);
     }
 }
 
 // The pipe client.
 pub struct PipeClient {
     /// Underlying libOS.
     libos: LibOS,
-    /// Rx pipe qd.
-    pipeqd_rx: QDesc,
-    /// Tx pipe qd.
-    pipeqd_tx: QDesc,
+    /// Pipe qd.
+    pipeqd: QDesc,
 }
 
 // Implementation of the pipe client.
 impl PipeClient {
     pub fn new(mut libos: LibOS, pipe_name: &str) -> Result<Self> {
-        // Open the Tx pipe. This is inverted from the server's perspsective, because the server reads from it's
-        // pipeqd_rx, but the clients writes to it.
-        let pipeqd_tx = match libos.open_pipe(&format!("{}:rx", pipe_name)) {
+        // Open the Tx pipe. This is inverted from the server's perspective, because the server reads from it's
```

Comment:
I believe we don't need this comment now, since there is only one pipeqd...?

Reply:
Sure.